sqlmesh/core/model/definition.py

```diff
@@ -1306,6 +1313,9 @@ def is_breaking_change(self, previous: Model) -> t.Optional[bool]:
         if not isinstance(previous, SeedModel):
             return None
 
+        if previous.fqn != self.fqn:
```

Comment:
why is this here and not something like partition by / cluster by changes oversight? maybe let's have a method with common checks?

Reply:
This is here because it feels appropriate to me to have an fqn change be considered a breaking change. If you think other things should also be considered breaking that currently aren't then that sounds reasonable but outside of the scope of this PR. It seems like a follow up task would be to check for all possible breaking properties that should be tracked, see what is common across models, and then create a method like you said of common checks. It just all seems outside of the scope of what I am doing here.